cypress-tests/cypress/support/e2e.js

```diff
@@ -24,3 +24,10 @@ import "@cypress/code-coverage/support";
 Cypress.on("uncaught:exception", (err, runnable) => {
   return false;
 });
+
+beforeEach(() => {
+  if (Cypress.env("deployment") === "proxy") {
```

Comment:
I think we can update this logic, by updating the browser properties.
Please check this later if it is time consuming.